config.cmake for YTL_ENABLE_IBV is missing INTERFACE in target_link_libraries

### Search before asking

- [x] I searched the [issues](https://github.com/alibaba/yalantinglibs/issues) and found no similar issues.


### What happened + What you expected to happen
run cmake with `-DYLT_ENBALE_IBV=ON`, error will occur, which is not expected
```
-- Enable ibverbs support
CMake Error at build/_deps/yalantinglibs-src/cmake/config.cmake:63 (target_link_libraries):
  INTERFACE library can only be used with the INTERFACE keyword of
  target_link_libraries
Call Stack (most recent call first):
  build/_deps/yalantinglibs-src/CMakeLists.txt:35 (include)
```
the error is owing to the missing interface keyword in `cmake/config.cmake`

```cmake
if (YLT_ENABLE_IBV)
    message(STATUS "Enable ibverbs support")
    if(CMAKE_PROJECT_NAME STREQUAL "yaLanTingLibs")
        add_compile_definitions("YLT_ENABLE_IBV")
        link_libraries(-libverbs)
    else ()
        target_compile_definitions(${ylt_target_name} INTERFACE "YLT_ENABLE_IBV") 
    target_link_libraries(${ylt_target_name} -libverbs) #missing INTERFACE here
    endif ()
endif ()


```


### Reproduction way
- link to yalantiniglibs
- run cmake with `-DYLT_ENBALE_IBV=ON`


### Anything else


### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

--- a/cmake/config.cmake
+++ b/cmake/config.cmake
@@ -60,7 +60,7 @@ if (YLT_ENABLE_IBV)
         link_libraries(-libverbs)
     else ()
         target_compile_definitions(${ylt_target_name} INTERFACE "YLT_ENABLE_IBV")
-        target_link_libraries(${ylt_target_name} -libverbs)
+        target_link_libraries(${ylt_target_name} INTERFACE -libverbs)
     endif ()
 endif ()
 

